workflow/tests/test_signals.py

```diff
@@ -268,17 +265,40 @@ def test_check_seats_save_team_exceed_notify(self):
                          'out!\nCurrent amount of editors: 2.\nSelected '
                          'amount of editors: 1.')
 
+    def test_check_seats_save_team_retrieve_subscription_fails(self):
+        """
+        The number of seats will be increased in the system but it's not
+        possible to check the quantity of the plan because the retrieve
+        failed.
+        """
+        self.tola_user.user.groups.add(self.group_org_admin)
+        self.tola_user.user.save()
+        self.org = Organization.objects.get(pk=self.org.id)
+        user = factories.User(first_name='John', last_name='Lennon')
+        tolauser = factories.TolaUser(user=user, organization=self.org)
+
+        json_obj = {
+            'message': "Sorry, we couldn't find that resource",
+            'error_code': 500
```

Comment:
Is it really 500 or 404?

Reply:
Neither 500 nor 404, it's not HTTP status code. It returns `resource_not_found` instead. I'll change it.